docs/GWDM/1.1.md

```diff
@@ -0,0 +1,680 @@
+
+## required
+
+required metadata needed for the GWDM
+        
+
+
+### gatewayId
+
+Need a field in Mauro that captures the datasetID to link to gateway database - or can we just use the one created in Mauro?
```

Comment:
I guess needs updating to not reference Mauro throughout?

Reply:
Good spot. This is some old nonsense that ive copied over from Damon at some point

I had a separate ticket for this here:
https://hdruk.atlassian.net/browse/GAT-3124